readme.md

```diff
@@ -2,22 +2,21 @@
 
 
 ## Environment(system) variables ##
+## Envirinoment variables should be sent in an .env file at the root of the project ##
 
 Variable  | Value
 ------------- | -------------
 **KAPITOL_TINMAN_PORT**  | **5444 or something**
-**KAPITOL_TINMAN_MODE**  | **development**
-**KAPITOL_TINMAN_SECRET_KEY**  | **anything**
+**KAPITOL_TINMAN_MODE**  | **dev**

```

Comment:
@greenac These variables have been renamed as,
TINMAN_PORT, TINMAN_MODE, and TINMAN_LOG_PATH